internal/txapp/txapp.go

```diff
@@ -517,6 +537,13 @@ func (r *TxApp) ProposerTxs(ctx context.Context, txNonce uint64) ([]*transaction
 			continue
 		}
 
+		// MaxTxBytes restrictions per block are enforced here
+		// From observations, the rlp encoded events size is almost always smaller than the event
```

Comment:
What is meant by "the rlp encoded events size is almost always smaller than the event"?  I get the RLP part, but what's size of just "the event" mean?

Reply:
of "the event"

Reply:
> of "the event"

Sorry, I still don't understand what it's saying